components/Profile/Interests.tsx

```diff
@@ -1,15 +1,15 @@
 import { Text, Tag } from "atomize"
 
-export const Interests = ({ interests }) => {
-    const skills = interests.length
-        ? interests.split(",").map((skill, id) => {
-              return (
+export const Interests = ({ interests}:{interests : string}) => {
+    const skills :JSX.Element[] = interests.length
```

Comment:
This line shows an ESLint error as follows:
>  4:19  error  'JSX' is not defined  (no-undef)

To fix this, add the `JSX` to the `globals` in the ESLint configuration file `.eslint.js` as follows:
```js
{
    ... // the lines above
    globals: {
        React: "writable",
        JSX: true,
    },
}
```

Reply:
Thanks. Fixed it.

Reply:
Great! Just one thing left, merge the changes in this repo to your branch or rebase using `git pull upstream main --rebase`. If you use rebase, you'll need to force push the changes.

Reply:
Yeah merged the changes.